sys/include/net/ng_ndp/types.h

```diff
@@ -0,0 +1,250 @@
+/*
+ * Copyright (C) 2015 Martine Lenders <mlenders@inf.fu-berlin.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    net_ng_ndp_types    Types for IPv6 neighbor discovery
+ * @ingroup     net_ng_ndp
+ * @brief       IPv6 neighbor discovery message types
+ * @{
+ *
+ * @file
+ * @brief       IPv6 neighbor discovery message type definitions
+ *
+ * @author  Martine Lenders <mlenders@inf.fu-berlin.de>
+ */
+#ifndef NG_NDP_TYPES_H_
+#define NG_NDP_TYPES_H_
+
+#include <inttypes.h>
```

Comment:
Yes, that's why I suggested `stdint.h` (which is required for this header). (Well, actually it's not really required to be included inside the header. If one really wants to keep includes out of any header, you only have to make sure that headers are included in C files in the right order, but that's probably not the path we want to take...)